fhd_core/beam_modeling/fhd_struct_init_antenna.pro

```diff
@@ -95,8 +99,9 @@ IF ~Keyword_Set(psf_dim) THEN $
     psf_dim=Ceil((Max(antenna.size_meters)*2.*Max(frequency_array)/speed_light)/kbinsize)
 psf_dim=Ceil(psf_dim/2.)*2. ;dimension MUST be even
 ;reset psf_dim if cetain conditions met.
-if keyword_set(debug_dim) then psf_dim=28.
+if keyword_set(debug_dim) then psf_dim=18.
```

Comment:
Probably should just remove debug dim altogether...

Reply:
done